Ghidra/Features/Base/src/main/java/help/screenshot/TutorialScreenShotGenerator.java

```diff
@@ -29,8 +29,7 @@ public TutorialScreenShotGenerator() {
 	}
 
 	/**
-	 * @deprecated use instead {@link #showImage(String)}, which will show a comparison of the old 
-	 * 				and new images
+	 * @deprecated
 	 * 
```

Comment:
showImage(String) no longer exists and doesn't appear to have been replaced. Since it's deprecated I just removed the comment.